test/metabase/test/data/impl/get_or_create.clj

```diff
@@ -147,9 +148,11 @@
     ;; Add DB object to Metabase DB
     (let [connection-details (tx/dbdef->connection-details driver :db database-definition)
           db                 (first (t2/insert-returning-instances! Database
-                                                                    :name    database-name
-                                                                    :engine  (u/qualified-name driver)
-                                                                    :details connection-details))]
+                                                                    (merge
```

Comment:
this is needed, otherwise the DB won't have scan field values schedule, as it previously rely on having a default value in the DB.